pyccl/__init__.py

```diff
@@ -34,6 +34,9 @@
 # BCM stuff
 from .bcm import bcm_model_fka
 
+# Old halo mass function
+from .massfunction import massfunc, halo_bias, massfunc_m2r
```

Comment:
You are missing sigmaM here.

Reply:
do we want to have two copies of sigmaM? Now it lives in power, but the API is still the same.

Reply:
The API includes function locations in the python package, so we have to.

Reply:
this doesn't make much sense to me. Users called `sigmaM` as `pyccl.sigmaM`, and they can keep doing so.
If we do this, we need to either change the name of the function in the new location or make it harder for users to call it (e.g. `pyccl.power.sigmaM`).

Reply:
Ahhh sorry. I mean we need a copy of sigmaM inside the massfunction module.

Reply:
ok, sorry, still confused: this will create two copies, one in power, one in massfunction. Both with the same signature, both imported in __init__. Won't this create problems?

Reply:
I think you can put `from .power import sigmaM` at the top of `massfunction.py` and this will work.

Reply:
you don't need to import into the init twice. I said the wrong thing above.

Reply:
we simply need a copy at `pyccl.massfunction.sigmaM`

Reply:
ok, let me check that

Reply:
that seems to work, done